src/index.js

```diff
@@ -1,5 +1,38 @@
-/* eslint-disable no-unused-vars */
-import _ from 'lodash';
+/* eslint-disable no-unused-vars, import/no-cycle */
+import _, { get } from 'lodash';
 import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getScores, newScore } from './js/api';
+
+const scoreBoard = document.querySelector('#scoresContainer');
+const refreshButton = document.querySelector('#refreshButton');
+const userName = document.querySelector('#userName');
+const userScore = document.querySelector('#userScore');
+const submitButton = document.querySelector('#submitScoreButton');
+
+function populateBoard(obj) {
```

Comment:
Please work on all the changes asked by the previous reviewer. I am unable to see the new score by clicking on the refresh button. YOu should use arrow functions instead of the `function` keyword. Please make sure everything works before asking for another code review.